### DISCUSSION TITLE
Is interrater reliability needed when following the SAFE procedure?

### DISCUSSION THREAD

**Discussion by Rianne-vD** [asreview/asreview#1907] [Q&A]
We use ASReview in several projects, and I wonder if and how to measure interrater reliability (IRR) when scanning the titles and abstracts. Although [the SAFE procedure](https://systematicreviewsjournal.biomedcentral.com/articles/10.1186/s13643-024-02502-7) is clearly explained, it does not mention anything about agreement between raters.

Given that ASReview presents new articles based on previous decisions, I assume that not all raters will see the same articles. This raises the question of how to effectively measure agreement in this context. I came across [another discussion on GitHub](https://github.com/asreview/asreview/discussions/1140) about this topic and [an example report](https://rpubs.com/langton_/asreview_report_example). Do you recommend the implementation of checking the IRR / agreement rate over a new batch of articles as an additional step between the training phase and the active learning phase as described in the SAFE procedure?

**Comment by FelixWdm** [comment]
Hi @Rianne-vD , the SAFE procedure has some steps to ensure 'safety' in the process, using one person to screen and ASReview to speed up the process. If you add more people to the process, there's the benefit of an extra set of eyes to verify these steps. But also the complexity that every additional person will potentially see different items and make different decisions. Using the datatools extension you can quickly get an overview of these differences. I hope this helps.

**Comment by bethgr** [comment]
Hi @Rianne-vD. Essentially, yes. I recommend checking IRR and using the example report is a nice tool. If/when raters do not see the same articles, we can see this as a missing data problem and the tool has additional options for calculating IRR with missing data.

**Reply by Rianne-vD** [reply]
Dear @bethgr, thank you for your reply. Would you recommend doing this in either the first or second step of the SAFE procedure, or both?

**Reply by bethgr** [reply]
Both when possible.

**Reply by MarielleDiepeveen** [reply]
Hi both, we are planning on working with ASReview as well and want to use the example report to determine the IRR during phase 2. What would your advice be, to run it during the screening, and/or at the end? 

And how to cope with an IRR that's (too) low during phase 2 (while being sufficient in phase 1)? Does the example report provide you with an overview of articles being labeled differently by the screeners, so you can discuss these?

**Reply by bethgr** [reply]
@MarielleDiepeveen It would be beneficial for the raters to calibrate a partial sample before proceeding. Yes, the example report does provide a really nice overview, along with some nice visuals.
